spec/controllers/user/swaps_controller_spec.rb

```diff
@@ -5,7 +5,8 @@
 
   context "when user has a potential swap" do
     let(:new_user) do
-      build(:user, id: 121, email: "foo@bar.com")
+      build(:user, id: 121, email: "foo@bar.com",
+      mobile_phone: build(:mobile_phone, user_id: 122, number: "07400 123456", verified: false))
```

Comment:
We need proper fixtures here really but I don't have time to mess with this right now!